src/assets/translations/szit.json

```diff
@@ -18511,7 +18511,7 @@
                   "verse": "8"
                 },
                 {
-                  "text": "Majd így folytatta: „Szépen semmibe veszitek Isten parancsát, hogy hagyományotokat állítsátok a helyére.”",
+                  "text": "Majd így folytatta: „Szépen semmibe veszitek Isten parancsát, hogy hagyományotokat állítsátok a helyére.",
```

Comment:
API response and text edition are aligned, but wrong.
https://szentiras.hu/api/ref/Mk7,9-13/SZIT

Multiple fixes are possible here, I think the PR proposal is the most straightforward.

Reply:
Documented